packages/internals/src/__tests__/getSchema.test.ts

```diff
@@ -34,20 +35,30 @@ async function testSchemaPath(fixtureName: string, schemaPathFromArgs?: string)
   }
 
   if (typeof asyncResult === 'string') {
-    asyncResult = toUnixPath(path.relative('.', asyncResult))
+    asyncResult = stripAnsi(toUnixPath(path.relative('.', asyncResult)))
   }
 
   if (asyncResult instanceof Error) {
-    asyncResult.message = toUnixPath(asyncResult.message.replace(__dirname, '.'))
+    asyncResult.message = stripAnsi(toUnixPath(asyncResult.message.replace(__dirname, '.')))
   }
 
   return asyncResult
 }
 
-it('returns null if no schema is found', async () => {
+it('throws error if schema is not found', async () => {
   const res = await testSchemaPath('no-schema')
 
-  expect(res).toMatchInlineSnapshot(`null`)
+  expect(res).toMatchInlineSnapshot(`
+    [Error: Could not find a schema.prisma file that is required for this command.
+    You can either provide it with --schema, set it as \`prisma.schema\` in your package.json or put it into the default location.
+    Checked following paths:
+
+    schema.prisma: file not found
+    prisma/schema.prisma: file not found
+    prisma/schema: directory not found
+
+    See also https://pris.ly/d/prisma-schema-location]
```

Comment:
Link should be updated to point to https://www.prisma.io/docs/orm/prisma-schema/overview/location I think.

Reply:
how would we do it?

Reply:
This is done in our dub.co account. Took care of it.